CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
         - Add initial update template on report sending, not creation.
         - Add option to set an emergency message on reporting pages.
         - Add label to questionnaire textarea. #3944
+        - Add CAPTCHA to contact form for non-UK IP addresses
```

Comment:
This only applies to UK things, so either goes in UK-specific part of changelog, or isn't mentioned at all.